tests/unit/DebugUtilsTest.ts

```diff
@@ -0,0 +1,546 @@
+        });
+    });
+    describe('getNumberOfLinesFromString', () => {
+        it('returns 1 when string is empty', () => {
+            expect(DebugUtils.getNumberOfLinesFromString('')).toBe(1);
+        });
+        it('returns 1 when no "\\n" are present in the string', () => {
+            expect(DebugUtils.getNumberOfLinesFromString('Something something something')).toBe(1);
+        });
+        it('returns k when there are k - 1 "\\n" present in the string', () => {
+            expect(DebugUtils.getNumberOfLinesFromString('Line1\n Line2\nLine3')).toBe(3);
+        });
+    });
+    describe('onyxDataToDraftData', () => {
+        it('returns an empty object if data is an empty object', () => {
+            expect(DebugUtils.onyxDataToDraftData({})).toEqual({});
+        });
+        it('returns a converted object where all values are string representations of the original values from data object', () => {
+            expect(
+                DebugUtils.onyxDataToDraftData({
+                    a: 1,
+                    b: 'a',
+                    c: [],
+                    d: {},
+                    e: true,
+                    f: false,
+                    g: [1],
+                    h: ['a'],
+                    i: [true],
+                    j: [false],
+                    k: [{}],
+                    l: [{a: 1, b: 'a', c: [], d: {}, e: true, f: false}],
+                    m: {a: 1, b: 'a', c: [], d: {}, e: true, f: false},
+                    n: undefined,
+                }),
+            ).toEqual({
+                a: '1',
+                b: 'a',
+                c: '[]',
+                d: '{}',
+                e: 'true',
+                f: 'false',
+                g: DebugUtils.onyxDataToString([1]),
+                h: DebugUtils.onyxDataToString(['a']),
+                i: DebugUtils.onyxDataToString([true]),
+                j: DebugUtils.onyxDataToString([false]),
+                k: DebugUtils.onyxDataToString([{}]),
+                l: DebugUtils.onyxDataToString([{a: 1, b: 'a', c: [], d: {}, e: true, f: false}]),
+                m: DebugUtils.onyxDataToString({a: 1, b: 'a', c: [], d: {}, e: true, f: false}),
+                n: 'undefined',
+            });
+        });
+    });
+    describe('validateNumber', () => {
+        it('does not throw SyntaxError when value is "undefined"', () => {
+            expect(() => {
+                DebugUtils.validateNumber('undefined');
+            }).not.toThrow();
+        });
+        it('does not throw SyntaxError when value is a valid string representation of a number', () => {
+            expect(() => {
+                DebugUtils.validateNumber('1');
+            }).not.toThrow();
+        });
+        it('throws SyntaxError when value contains white spaces', () => {
+            expect(() => {
+                DebugUtils.validateNumber('    ');
+            }).toThrow();
+        });
```

Comment:
Do we really need to cover this? Isn't the same as `throws SyntaxError when value is not a valid number`?